DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -3207,6 +3207,8 @@ private static void UpdateFoldersForParentId()
 
         private static void UninstallPackage(string packageName, string packageType, bool deleteFiles = true, string version = "")
         {
+	    DnnInstallLogger.InstallLogInfo(string.Concat(Localization.Localization.GetString("LogStart", Localization.Localization.GlobalResourceFile), "Package:", packageName, " Type:", packageType, " Version:", version));

```

Comment:
Could we add the phrase "uninstall" at some point in this.  Right now this would result in an output of

Start Package {Name} Type: {Type} Version: {Version}

Based on the value of "LogStart" being simply the word start.